api/http/handler/registries/handler.go

```diff
@@ -46,6 +46,9 @@ func NewHandler(bouncer *security.RequestBouncer) *Handler {
 		bouncer.AuthorizedAccess(httperror.LoggerHandler(h.registryDelete))).Methods(http.MethodDelete)
 	h.PathPrefix("/registries/{id}/v2").Handler(
```

Comment:
What do you think about renaming this API endpoint to `/registries/{id}/proxies/v2` or `/registries/{id}/proxies/docker` for consistency?

Reply:
As we are building `/registries/{id}/proxies/gitlab/api/v4` and stripping `/registries/{id}/proxies/gitlab` for gitlab, `/registries/{id}/proxies/docker` sounds good.

We will build `/registries/{id}/proxies/docker/v2` and strip `/registries/{id}/proxies/docker` for docker native registry API

Reply:
Sounds good to me.

Reply:
Done.